rustls/src/lib.rs

```diff
@@ -383,6 +383,7 @@ pub use crate::ticketer::Ticketer;
 pub use crate::tls12::Tls12CipherSuite;
 pub use crate::tls13::Tls13CipherSuite;
 pub use crate::versions::{SupportedProtocolVersion, ALL_VERSIONS, DEFAULT_VERSIONS};
+pub use enums::*;
```

Comment:
I prefer to never use `*` imports, as they make it less obvious where names come from.